Packaging.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <PreReleaseLabel>preview2</PreReleaseLabel>
     <PackageDescriptionFile>$(ProjectDir)pkg/descriptions.json</PackageDescriptionFile>
-    <PackageLicenseFile>$(ProjectDir)pkg/dotnet_library_license.txt</PackageLicenseFile>
+    <PackageLicenseFile>$(ProjectDir)LICENSE</PackageLicenseFile>
     <PackageThirdPartyNoticesFile>$(ProjectDir)pkg/ThirdPartyNotices.txt</PackageThirdPartyNoticesFile>
```

Comment:
Does ThirdPartyNotices need to get onto the same plan? (Reference the one in the repo root, not a separate out-of-sync copy.)